tests.py

```diff
@@ -1165,8 +1144,6 @@ class TestBzr(VCSMixin, unittest.TestCase):
     vcs = BzrHelper()
 
 
-@unittest.skipIf(HAS_OEM_CODEC,
-                 "Python 3.6 lets us use 'oem' codec instead of guessing")
 class TestBzrTerminalCharsetDetectionOnOldPythons(unittest.TestCase):
```

Comment:
I don't think these two test classes could possibly work at the same time.  This one is meant for Python < 3.6 and should be removed entirely.